spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1106,8 +1106,24 @@ For the conversion of inbound messages, especially when the target is a POJO, th
 
 Besides the conversions that it supports out of the box, Spring Cloud Stream also supports registering your own message conversion implementations.
 This allows you to send and receive data in a variety of custom formats, including binary, and associate them with specific `contentTypes`.
-In order to do so, you can create a class that extends `AbstractMessageConverter`
+In order to do so, you can create a class of type `org.springframework.messaging.converter.MessageConverter` and make it available in the application context.
```

Comment:
I believe that itself should be documented clearly (the role of `@StreamListener` as compared-to/combined-with `content-type`)